src/main/java/JP2ImageConverter/services/ImagePreproccessingService.java

```diff
@@ -168,11 +168,11 @@ public String convertCr2(String fileName) throws Exception {
     }
 
     /**
-     * Run Exiftool to convert NEF images to JPEG
-     * @param fileName an image file
+     * Run Exiftool to convert NEF and NRW images to JPEG
+     * @param fileName an NEF or NRW image file
      * @return temporaryFile a temporary JPEG file
      */
-    public String convertNef(String fileName) throws Exception {
+    public String convertNefAndNrw(String fileName) throws Exception {
```

Comment:
It might make sense to name these base on what they do rather than what their expected input formats are, so we don't need to rename them when we find additional formats that use the method. Like I think this is the only method using exiftool convert, so it could be `convertToJpgWithExiftool`